packages/components/src/components/page-overlay/page-overlay.scss

```diff
@@ -1,4 +1,8 @@
 .dc-page-overlay {
+    display: flex;
```

Comment:
Possible to wrap this in `@include mobile`? Since it's best we avoid modifying default rules on desktop in our changes. (Although I did have a quick look and it doesn't seem to have any issues).

Reply:
Sure 👍